build/build/src/enso.rs

```diff
@@ -223,7 +223,9 @@ impl BuiltEnso {
         if errors.is_empty() {
             Ok(())
         } else {
-            error!("{} test suit(s) failed.", errors.len());
+            let summary = errors.as_slice().iter().map(|e| e.to_string()).collect::<Vec<_>>().join(", ");
+            println!("::error title=Failed Standard Library Tests::{} test suite(s) failed: {}", errors.len(), summary);
```

Comment:
I see a line starting with `::` - that must be a _GitHub Actions command line_! Where is the syntax documented?

Reply:
I am glad we ended up doing the same - e.g. using `println!` to print the command line without a prefix. Just like at [my PR](https://github.com/enso-org/enso/pull/10914/files#r1738221228).